src/components/Avatar/Avatar.tsx

```diff
@@ -49,14 +52,21 @@ function Avatar({
   style,
   title,
   userName,
+  plainTooltip = false,
   ...rest
 }: Props) {
   const [error, setError] = useState(false);
 
-  const altText = `Avatar for ${userName}`;
+  const altText = plainTooltip || !userName ? "Profile Image" : `Avatar for ${userName}`;
 
   return error || !src ? (
-    <UserIcon className={`${className} --${size || ""}`} description={altText} style={style} />
+    <UserIcon
+      userName={userName}
+      plainTooltip={plainTooltip}
+      className={`${className} --${size || ""}`}
+      description={altText}
+      style={style}
+    />
```

Comment:
`userName` is not being passed to UserIcon component, that way we will never have the condition to not show the plainTooltip valid. Can you update to pass the userName here?

Reply:
> `userName` is not being passed to UserIcon component, that way we will never have the condition to not show the plainTooltip valid. Can you update to pass the userName here?

@Okisa username is added in line 64 by me already. In older version it was not there due to which it always showed usericon for user. This fixes that too. Let me know if I’m missing something.